packages/frontend/src/components/Buttons/ConnectWalletButton.tsx

```diff
@@ -1,13 +1,82 @@
+import { CoinbaseWalletSDK } from '@coinbase/wallet-sdk'
+import Portis from '@portis/web3'
 import { useEthers } from '@usedapp/core'
+import WalletConnectProvider from '@walletconnect/web3-provider'
+import { CONFIG } from 'src/config/config'
+import Web3Modal from 'web3modal'
 
 import { Button, ButtonProps } from './Button'
 
 type ConnectWalletButtonProps = Omit<ButtonProps, 'onClick' | 'children'>
 
 export const ConnectWalletButton = (props: ConnectWalletButtonProps) => {
```

Comment:
The page does not remember the last wallet used. When I refresh I need to press connect wallet again.